test/export/CodeSystemExporter.test.ts

```diff
@@ -247,15 +246,15 @@ describe('CodeSystemExporter', () => {
     rule2.caretPath = 'count';
     rule2.value = 5;
     codeSystem.rules.push(rule2);
-    codeSystem.concepts = [new FshConcept('myCode'), new FshConcept('anotherCode')];
+    codeSystem.rules.push(...[new ConceptRule('myCode'), new ConceptRule('anotherCode')]);
     doc.codeSystems.set(codeSystem.name, codeSystem);
     const exported = exporter.export().codeSystems;
     expect(exported.length).toBe(1);
     expect(exported[0].count).toBe(5);
     expect(loggerSpy.getAllLogs('warn').length).toBe(0);
   });
 
-  it('should warn when ^count does not match number of concepts in #complete CodeSystem', () => {
+  it('should warn when ^count does not match number of rules in #complete CodeSystem', () => {
```

Comment:
This description isn't exactly right -- because `^count` should correspond to the number of concepts, not rules.  Now that `CodeSystem` supports `include` rules, it's possible for a `CodeSystem` to have only one rule but have a `count` > 1.

Can you fix the description _and_ add a test to ensure the proper count when using `insert` rules?

Reply:
Oh that looks like a find and replace typo, but it helped you think of a test case, so good job find and replace.